bot.js

```diff
@@ -5,8 +5,13 @@ const {MAPLE_STORY_CLASSES} = require("./constants.json")
 const client = new Discord.Client();
 
 /* importing functions from the commands dir */
-var add = require('./commands/add.js');
-var remove = require('./commands/remove.js');
+const add = require('./commands/add.js');
+const remove = require('./commands/remove.js');
+const postRoster = require('./commands/postRoster.js');
```

Comment:
Is this postRoster instead of roster to avoid naming collisions with roster being used?

Reply:
kind of yea. Later on, I want to use a global roster variable since each function uses a let roster = getRoster() and having that global will speed the bot up a bunch.

TLDR: postRoster is to avoid naming collision of the future global variable we don't have yet.